pylib/openrazer/client/fx.py

```diff
@@ -68,6 +68,46 @@ def __init__(self, serial: str, capabilities: dict, daemon_dbus=None, matrix_dim
 
         self.misc = MiscLighting(serial, capabilities, self._dbus)
 
+    @property
+    def effect(self) -> str:
+        """
+        Get current effect
+
+        :return: Effect name
+        :rtype: str
+        """
```

Comment:
Please add example strings to the doc

Reply:
Please explain I am not familiar with Python

Reply:
Sorry for not replying earlier, I meant `:return: Effect name (e.g. "spectrum")` for example, just so the user has an idea what the return value looks like without running the call

Reply:
Done